test/lib/creation.go

```diff
@@ -373,8 +374,12 @@ func (c *Client) CreatePodOrFail(pod *corev1.Pod, options ...func(*corev1.Pod, *
 			c.T.Fatalf("Failed to configure pod %q: %v", pod.Name, err)
 		}
 	}
-	c.T.Logf("Creating pod %+v", pod)
-	if _, err := c.Kube.CreatePod(pod); err != nil {
+	err := reconciler.RetryUpdateConflicts(func(attempts int) (err error) {
```

Comment:
Just wanted to say this is an awesome reuse of this function!

cc @markusthoemmes